### PR TITLE
Appeals - Remove appeals when switch account list

### PR DESCRIPTION
**Pull request by dr-bizz**
## Description
Ensure React keeps track of the appeals, to ensure they are removed correctly when the user switches account.

## Checklist:

- [ ] I have given my PR a title with the format "MPDX-(JIRA#) (summary sentence max 80 chars)"
- [ ] I have applied the appropriate labels. (_Add the label "On Staging" to get the branch automatically merged into staging._)
- [ ] I have requested a review from another person on the project

### CODE REVIEW

**Review by caleballdrin — APPROVED** [review]
That fixes the issue for me!

**Comment by dr-bizz** [comment]
Great!

File: src/components/Tool/Appeal/InitialPage/Appeals.tsx [hunk 1]
```diff
@@ -92,13 +92,11 @@ const Appeals: React.FC<AppealsProps> = ({ accountListId }) => {
...
         </>
       ) : data?.regularAppeals.nodes.length ? (
-        <>
-          {data.regularAppeals.nodes.map((appeal) => (
-            <Box key={appeal.name} mb={3}>
-              <Appeal appeal={appeal} changePrimary={handleChangePrimary} />
-            </Box>
-          ))}
-        </>
+        data.regularAppeals.nodes.map((appeal) => (
```
Comment (canac) [inline src/components/Tool/Appeal/InitialPage/Appeals.tsx:95]: @dr-bizz Do you know why moving it outside of the fragment changes things?

Comment (dr-bizz) [reply src/components/Tool/Appeal/InitialPage/Appeals.tsx:95]: The issue was the `key`. At first, it was the appeal's name, which sometimes had duplicates. I switched it to the `id`, which would be unique.

I don't think removing the fragment did anything apart from simplifying the code (in my opinion). Maybe React read the Fragment as an item and kept it in the render, but that is unlikely.

Comment (canac) [reply src/components/Tool/Appeal/InitialPage/Appeals.tsx:95]: Gotcha. I missed that the `key` changed. That's probably what fixed it.

**Pull request merged**